src/components/ble/weather/WeatherService.h

```diff
@@ -36,15 +36,11 @@
 int WeatherCallback(uint16_t connHandle, uint16_t attrHandle, struct ble_gatt_access_ctxt* ctxt, void* arg);
 
 namespace Pinetime {
-  namespace System {
-    class SystemTask;
-  }
-
   namespace Controllers {
 
     class WeatherService {
     public:
-      explicit WeatherService(System::SystemTask& system, DateTime& dateTimeController);
+      explicit WeatherService(DateTime& dateTimeController);
```

Comment:
Unrelated to this PR, but does the weather service really need a non-const dateTimeController? Should the weather service be able to modify the system time?